code/datums/tutorial/marine/hospital_corpsman_sandbox.dm

```diff
@@ -0,0 +1,682 @@
+	message_to_player("Remember, on the field, immediate resupply will not always be possible! You won't know for certain when your next chance will arrive, so stock up while you can!")
+	message_to_player("When you are ready, leave the supply room, then click the 'Ready Up' action on the top left of your screen to begin your next round.")
+
+	give_action(tutorial_mob, /datum/action/hm_tutorial/sandbox/ready_up, null, null, src)
+
+/datum/tutorial/marine/hospital_corpsman_sandbox/proc/spawn_agents()
+	SIGNAL_HANDLER
+
+	for(var/i in 1 to (round(rand(min_survival_agents, max_survival_agents))))
+		var/mob/living/carbon/human/realistic_dummy/active_agent = new(agent_spawn_location)
+		arm_equipment(active_agent, /datum/equipment_preset/uscm/tutorial_rifleman)
+		var/turf/dropoff_point = loc_from_corner(round(rand(6, 8), 1), round(rand(1, 3)))	// Picks a random turf to move the NPC to
+		agents[active_agent] = dropoff_point
+		active_agent.a_intent = INTENT_DISARM
+		simulate_condition(active_agent)
+		var/obj/item/clothing/suit/storage/marine/medium/armor = active_agent.get_item_by_slot(WEAR_JACKET)
+		RegisterSignal(armor, COMSIG_ITEM_UNEQUIPPED, PROC_REF(item_cleanup))
+
+	addtimer(CALLBACK(src, PROC_REF(eval_agent_status)), 3 SECONDS)	// Gives time for NPCs to pass out or die, if their condition is severe enough
+	if((survival_difficulty >= TUTORIAL_HM_INJURY_SEVERITY_FATAL) && (rand() <= 0.75))	// If above difficulty FATAL, starts a random timer to spawn a booboo agent
+		booboo_timer = addtimer(CALLBACK(src, PROC_REF(eval_booboo_agent)), (rand(15,25)) SECONDS, TIMER_STOPPABLE)
+
+/datum/tutorial/marine/hospital_corpsman_sandbox/proc/simulate_condition(mob/living/carbon/human/target)
+	SIGNAL_HANDLER
+
+	// Simulates patient NPC injuries
+
+	var/damage_amount_split = ((round(rand(1, 100))) / 100)	// How damage should be split between brute and burn
+	var/list/limbs = target.limbs
+	var/amount_of_parts = round(rand(1, 6))	// Amount of times to roll for a limb fracture
+	var/patient_type = pick(75;PATIENT_TYPE_MUNDANE, 15;PATIENT_TYPE_ORGAN, 10;PATIENT_TYPE_TOXIN) // 75% chance for mundane damage, 15% for organ damage, 10% for toxin
+
+	if(patient_type >= PATIENT_TYPE_MUNDANE)
+		for(var/i in 1 to amount_of_parts)
+			var/obj/limb/selected_limb = pick(limbs)
+			var/damage_amount = (round(rand((40 * survival_difficulty), (50 * survival_difficulty))))
+			selected_limb.take_damage(round((damage_amount * damage_amount_split) / amount_of_parts), round((damage_amount * (1 - damage_amount_split)) / amount_of_parts))
+			if((damage_amount > 30) && (rand()) < (survival_difficulty / 10))
+				selected_limb.fracture()
+	if(patient_type == PATIENT_TYPE_ORGAN)	// applies organ damage AS WELL as mundane damage if type 2
+		var/datum/internal_organ/organ = pick(target.internal_organs)
+		target.apply_internal_damage(round(rand(1,(survival_difficulty*3.75))), "[organ.name]")
```

Comment:
when given a lower and upper bound rand() returns an integer so round() is not needed https://www.byond.com/docs/ref/#/proc/rand